prow/apis/prowjobs/v1/types.go

```diff
@@ -164,7 +164,7 @@ type ProwJobSpec struct {
 	ReporterConfig *ReporterConfig `json:"reporter_config,omitempty"`
 
 	// RerunAuthConfig holds information about which users can rerun the job
-	RerunAuthConfig RerunAuthConfig `json:"rerun_auth_config,omitempty"`
```

Comment:
The fix makes perfect sense, did you check that there are no other places where we access the `RerunAuthConfig` and would now need a nilcheck?

Also small nit: three commits for chaing 23 LOC is probably a bit too much

Reply:
thanks, checking...